src/main/java/com/epam/flightscontroltool/boundary/exception/CompanyExceptionHandler.java

```diff
@@ -1,30 +1,30 @@
-package com.epam.flightscontroltool.controller.exception.airplane;
+package com.epam.flightscontroltool.boundary.exception;
 
-import com.epam.flightscontroltool.controller.exception.ErrorMessage;
 import lombok.extern.slf4j.Slf4j;
+import org.springframework.core.Ordered;
+import org.springframework.core.annotation.Order;
 import org.springframework.http.HttpStatus;
 import org.springframework.web.bind.annotation.ControllerAdvice;
 import org.springframework.web.bind.annotation.ExceptionHandler;
 import org.springframework.web.bind.annotation.ResponseBody;
 import org.springframework.web.bind.annotation.ResponseStatus;
-import org.springframework.web.context.request.WebRequest;
 import org.springframework.web.servlet.mvc.method.annotation.ResponseEntityExceptionHandler;
 
-import java.util.Date;
+import java.time.LocalDateTime;
 
 @Slf4j
 @ControllerAdvice
+@Order(Ordered.HIGHEST_PRECEDENCE)
 @ResponseBody
 public class CompanyExceptionHandler extends ResponseEntityExceptionHandler {
```

Comment:
For this kind of project I would add only one Exception Handler with all methods. Now you have many Exception Handlers with one method in each of them.